model-server-test/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:9.6
```

Comment:
Any reason for using such an old postgres version? Looks like we should be at least on version 12: https://endoflife.date/postgresql

Reply:
This comment is still valid for the compose file used in the tests.

Reply:
That's why: https://github.com/modelix/modelix.kubernetes/blob/main/db/Dockerfile#L1

Reply:
Updated it to 16.2